python/nav/models/profiles.py

```diff
@@ -83,9 +84,10 @@
 ### Account models
 
 
-class Account(models.Model):
+class Account(AbstractBaseUser):
     """NAV's basic account model"""
 
+    USERNAME_FIELD = 'login'
```

Comment:
Also, EMAIL_FIELD. It is better to copy the power of AbstractUser than AbstractBaseUser.

Reply:
Turns out django-allauth might not care about this field, more research needed.

Reply:
Ok, e-mail field added :)

Reply:
django-auth-ldap has a way of copying ldap attrs to the django user so that's good.